frontend/src/metabase/lib/array.ts

```diff
@@ -0,0 +1,5 @@
+export function moveElement<T>(array: T[], oldIndex: number, newIndex: number) {
```

Comment:
We have similar utilities in `core/utils`, that looks like something that should go there

Reply:
@ranquild added to `metabase/core/utils/arrays` — was trying to follow the existing naming conventions in there (with `types` and `errors` folders). Looks good?